docs/src/main/sphinx/release/release-370.md

```diff
@@ -0,0 +1,62 @@
+# Release 370 (4 Feb 2022)
+
+## General
+
+* Automatically raise the operating system maximum limit for open files to the
+  recommended value in the Trino init script if required. ({issue}`8819`)
+* Add support for `DEFAULT` keyword in `ALTER TABLE...SET PROPERTIES...`.
+  ({issue}`10331`)
+* Improve performance of map and row types. ({issue}`10469`)
+* Fix query scheduling regression introduced in Trino 360 that caused
+  coordinator slowness in assigning splits to workers. ({issue}`10839`)
+* Fix `information_schema` query failure when a `IS NOT NULL` predicate is used.
+  ({issue}`10861`)
+
+## Security
+
+* Fix server start failure when using JWT and OAuth 2.0 authentication together
+  (`http-server.authentication.type=jwt,oauth2`). ({issue}`10811`)
+
+## Docker image
+
+* Change base image to `registry.access.redhat.com/ubi8/ubi`, since CentOS 8 has
+  reached end-of-live. ({issue}`10866`)
```

Comment:
```suggestion
  reached end-of-life. ({issue}`10866`)
```

Reply:
haha ... I fixed it from the original proposal of end-of-line .. and messed that up.